From f7269add43cb0ad2cfcb1db7aedbe0c0924e2dbf Mon Sep 17 00:00:00 2001
From: Sergei Makarov <serg.makarov@samsung.com>
Date: Fri, 5 May 2023 15:22:32 +0300
Subject: [PATCH 19/29] [secndp, pytorch] Don't mount `/dev/axdimm_device`

This device is not present in simulator setting, and
mounting it raises an error. To my knowledge, Gramine
doesn't work with physical device yet anyway, so this
mount can be safely removed.

Signed-off-by: Sergei Makarov <serg.makarov@samsung.com>
---
 application/axdimm_secndp_test/secndp.manifest.template | 2 --
 application/pytorch/conf.pytorch.manifest.template      | 2 --
 2 files changed, 4 deletions(-)

diff --git a/application/axdimm_secndp_test/secndp.manifest.template b/application/axdimm_secndp_test/secndp.manifest.template
index 2908c146..d2e0a5a2 100644
--- a/application/axdimm_secndp_test/secndp.manifest.template
+++ b/application/axdimm_secndp_test/secndp.manifest.template
@@ -18,7 +18,6 @@ fs.mounts = [
   { uri = "file:{{ arch_libdir }}", path = "{{ arch_libdir }}" },
   { uri = "file:/usr", path = "/usr" },
   { uri = "file:/etc", path = "/etc" },
-  { uri = "dev:/dev/axdimm_device", path = "/dev/axdimm_device" },
   { uri = "file:{{ pnm_driver }}/test", path = "{{ pnm_driver }}/test" },
   { uri = "file:{{ pnm_driver }}/ai", path = "{{ pnm_driver }}/ai" },
   { uri = "file:{{ pnm_driver }}/secure", path = "{{ pnm_driver }}/secure" },
@@ -41,7 +40,6 @@ sgx.allowed_files = [
 ]
 
 sgx.trusted_files = [
-  "file:/dev/",
   "file:{{ gramine.libos }}",
   "file:{{ entrypoint }}",
   "file:{{ gramine.runtimedir() }}/",
diff --git a/application/pytorch/conf.pytorch.manifest.template b/application/pytorch/conf.pytorch.manifest.template
index 45004dce..ed3e4dcf 100644
--- a/application/pytorch/conf.pytorch.manifest.template
+++ b/application/pytorch/conf.pytorch.manifest.template
@@ -24,7 +24,6 @@ fs.mounts = [
   { uri = "file:{{ dlrm_data_path }}", path = "/input" },
   { type = "tmpfs", path = "/tmp" },
   { type = "shm" },
-  { uri = "dev:/dev/axdimm_device", path = "/dev/axdimm_device" },
   { uri = "file:/dev/pnm/", path = "/dev/pnm/" },
   { uri = "file:/sys/class", path = "/sys/class", type="chroot" },
 ]
@@ -36,7 +35,6 @@ sgx.debug = true
 sgx.insecure__enable_shared_range = "true"
 
 sgx.trusted_files = [
-  "file:/dev/",
   "file:/{{ dlrm_path }}/",
   "file:{{ gramine.libos }}",
   "file:{{ entrypoint }}",
-- 
2.34.1

